team_production_system/serializers/availability.py

```diff
@@ -29,34 +35,28 @@ def create(self, validated_data):
         # Check if end time is between a start time and end time of
         # an existing availability
         overlapping_condition2 = Availability.objects.filter(
```

Comment:
Instead of whitelisting conditions1, let's change it to condition_1 since condition passes the spellcheck